ui_framework/components/panel/_panel.scss

```diff
@@ -1,11 +1,18 @@
 .kuiPanel {
-  border: $globalBorderThick;
+  border: $globalBorderThin;
+  border-radius: $globalBorderRadius;
 }
 
 .kuiPanel--withHeader {
+  // border-top: none;
```

Comment:
I think we can delete this class.

Also, we need to replace `kuiPanel--withHeader` with `kuiPanel--withToolbar` in two places:
* Line 15 of `src/core_plugins/kibana/public/management/landing.html`
* Line 1 of `src/ui/public/table_info/table_info.html`